containers/ome-tiff-offsets/test-output-expected/multi-channel.json

```diff
@@ -0,0 +1 @@
+{"offsetValues": [8, 2712, 4394]}
```

Comment:
@ilan-gold : This is what you wanted my opinion on? Looks fine, but if you really just need the list, maybe...?
```suggestion
[8, 2712, 4394]
```
and rename to `offsets.json`?

Reply:
Would you be ok with `filename.offsets.json` from `filename.ome.tif`?  I don't want to have to specify a directory structure here and the names will start colliding if it is just `offsets.json`.

Reply:
Sorry: didn't realize the name collision problem. Yeah: just wanted to have something about offsets in the filename. Thanks!